ipol_demo/modules/core/core.py

```diff
@@ -2088,20 +2092,16 @@ def delete_demo(self, demo_id):
             if not isinstance(result, Ok):
                 error_message += f"Error when removing demoextras: {result.value} \n"
 
-            if (
-                self.post("/api/blobs/delete_demo", data=userdata).json()["status"]
-                != "OK"
-            ):
-                error_message += "API call /blobs/delete_demo failed.'\n"
+            response, _ = self.post("/api/blobs/delete_demo", "post", data=userdata)
+            if response.json()["status"] != "OK":
+                error_message += f"API call /blobs/delete_demo failed.'\n"
 
             # delete the archive
-            res_archive = self.post("/api/archive/delete_demo", data=userdata).json()
-            if (
-                res_archive["status"] != "OK"
-                and res_archive["message"] != "No archives found"
-            ):
-                error_message += f"API call /api/archive/delete_demo failed. {demo_id}"
-
+            _, status_code = self.post(f"/api/archive/demo/{demo_id}", "delete")
+            if status_code == 404:
+                error_message += (
```

Comment:
In line 2097 we perform error_message += <string>, but in line 2102 we append a tuple.
This seems contradictory or at least unneeded.

Reply:
`("<string>")` is not a tuple in Python

Reply:
My question is why is written as a tuple in 2102, when in the other assignment it's just a string concatenation. You can write the string as a tuple of characters (and the type will be a string), but why not simply concatenating in 2102?

Reply:
The statement is split over multiple lines with `()` because of the length of the line otherwise. There is nothing related to tuples here.

Reply:
I see, just line breaking. Thanks.